File: Operations_vb/Classes/Customer.vb
Imports Interfaces

Namespace Classes
    Public Class Customer
        Implements IBase

        Public ReadOnly Property Id() As Integer Implements IBase.Id
        Public Property CustomerIdentifier() As Integer
        Public Property CompanyName() As String
        Public Property ContactIdentifier() As Integer?
        Public Property ContactTypeIdentifier() As Integer?
        Public Property Street() As String
        Public Property City() As String
        Public Property PostalCode() As String
        Public Property CountryIdentfier() As Integer?
        Public Property Phone() As String
        Public Property ModifiedDate() As DateTime?
        Public Property InUse() As Boolean?

    End Class

End Namespace

File: Operations/Classes/Customer.cs
using System;
using Operations.Interfaces;

namespace Operations.Classes
{
    public class Customer : IBase
    {
        public int Id { get; }
        public int CustomerIdentifier { get; set; }
        public string CompanyName { get; set; }
        public int? ContactIdentifier { get; set; }
        public int? ContactTypeIdentifier { get; set; }
        public string Street { get; set; }
        public string City { get; set; }
        public string PostalCode { get; set; }
        public int? CountryIdentfier { get; set; }
        public string Phone { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public bool? InUse { get; set; }

    }

}
